tests/framework/components/mocks.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, *args, name="mock_component_a"):
         self.args = args
         self.builder_used_for_setup = None
 
+    def create_lookup_tables(self, builder):
```

Comment:
Overwriting this method in these two mock components to handle how we mock builder in some tests.